src/components/v5/common/ActionSidebar/partials/ActionSidebarContent/ActionSidebarContent.tsx

```diff
@@ -51,25 +52,25 @@ const MSG = defineMessages({
 
 const ActionSidebarFormContent: FC<ActionSidebarFormContentProps> = ({
   getFormOptions,
-  isMotion,
+  action: actionData,
 }) => {
-  const { formatMessage } = useIntl();
-
+  const { readonly } = useAdditionalFormOptionsContext();
   const { formComponent: FormComponent, selectedAction } =
     useSidebarActionForm();
-  const descriptionMetadata = useActionDescriptionMetadata();
-  const { readonly } = useAdditionalFormOptionsContext();
 
+  const descriptionMetadata = useActionDescriptionMetadata(actionData);
   const {
     setValue,
     formState: {
+      isSubmitting,
       errors: { title: titleError },
     },
   } = useFormContext();
-
   const { noPermissionsError } = usePermissionsValidation();
   const { noReputationError } = useReputationValidation();
 
+  useDisableFormOnSubmit();
```

Comment:
Instead of a separate hook, could this be an option on `ActionForm` component for example?

Reply:
Moved to `Form` component